packages/@textlint/textlint-plugin-text/src/index.ts

```diff
@@ -1,6 +1,6 @@
 // LICENSE : MIT
 "use strict";
 import { TextProcessor } from "./TextProcessor";
-module.exports = {
+export const textPlugin = {
```

Comment:
Use default export. It is public API.
We can not change this because it does breaking change.

We prefer to use named import/export internally, But Public API is not that.

Reply:
```suggestion
export default {
```

Reply:
textlint plugin need to export as default.
It is convetional rule.

https://textlint.github.io/docs/plugin.html

Reply:
Oh, I'm sorry. I try to revert commit.

Reply:
textlint (textlint/packages/textlint) test:unit failed when revert to default export, so I fix `require` to `import` in textlint-core.ts
e26a747

Reply:
https://github.com/textlint/textlint/blob/953eb036d4a4b6a3249a2eaaa9af27e4da7b69ea/packages/%40textlint/kernel/src/descriptor/TextlintPluginDescriptor.ts#L33-L36

I also found this error message may need need to correct using `export default`.